gpkit/tests/t_model.py

```diff
@@ -448,13 +448,42 @@ def __init__(self, n, **kwargs):
         Model.__init__(self, None, [a >= c/b], **kwargs)
 
 
+class Box(Model):
+    "simple box for model testing"
+    def __init__(self):
+        h = Variable("h", "m", "height")
+        w = Variable("w", "m", "width")
+        d = Variable("d", "m", "depth")
+        Model.__init__(self, 1/(h*w*d), [])
+
+class BoxAreaBounds(Model):
+    "for testing functionality of separate analysis models"
+    def __init__(self, box):
+        A_wall = Variable("A_{wall}", 100, "m^2", "Upper limit, wall area")
+        A_floor = Variable("A_{floor}", 50, "m^2", "Upper limit, floor area")
+
+        constraints = [2*box["h"]*box["w"] + 2*box["h"]*box["d"] <= A_wall,
+                       box["w"]*box["d"] <= A_floor]
+
+        Model.__init__(self, None, constraints)
+
+
```

Comment:
@arthur-brown, fyi, this commit was inspired by your simple box example that fails on master but works on this branch